### DISCUSSION TITLE
Pages site still showing a 404 after following documentation and troubleshooting.

### DISCUSSION THREAD

**Discussion by WHHWII** [community/community#127948] [Other Feature Feedback, Questions, & Ideas] [answered]
# Question
Hello, I have been trying to create a github pages site for my portfolio, however for some reason, it still only shows a 404. I have followed and looked over the tutorial/documentation several times, as well as the common troubleshooting steps provided by github, stackoverflow, and freinds in real life, and have still been unable to get it to show anything but a 404. Any help would be greatly appreciated. 
# Images
Here's some screenshots just showing the basic stuff i assume people will ask me to check first. Not saying I _didn't_ do anything wrong here, but I've looked over them over and over and haven't been able to figure out what I've messed up.
![image](https://github.com/community/community/assets/36775534/bb47d823-4ab4-4660-9fba-d03399761571)
![image](https://github.com/community/community/assets/36775534/989f5ba5-9662-4f6a-a82c-08216640201d)
![image](https://github.com/community/community/assets/36775534/4273c606-7990-4457-8590-836913e8d91f)
![image](https://github.com/community/community/assets/36775534/b0a90fec-9d08-412a-86be-e0987e5684bf)
![image](https://github.com/community/community/assets/36775534/1f4de4be-b607-4ee8-976f-81bda90ae8f8)
Chrome:
![image](https://github.com/community/community/assets/36775534/fca104c3-f47c-405c-bd27-410f24d69c8b)
Firefox:
![image](https://github.com/community/community/assets/36775534/d184b0d0-617b-4fad-b1fe-55289505bf1a)

**Comment by davevad93** [comment, marked as answer]
Hi @WHHWII ,

GitHub Pages is case sensitive, try changing `Index.html` to `index.html` in lowercase, I'm pretty sure that's the issue.

**Reply by WHHWII** [reply]
This was it. Thanks so much,

**Reply by davevad93** [reply]
You're welcome @WHHWII !
